### PR TITLE
prerelease updates doc for 2.4.2, various row and column article updates and revisions

### PR DESCRIPTION
**Pull request by nancyhild**
New article for the new prelease updates setting in 2.4.2 plus some changes and updates for row and column docs.

### CODE REVIEW

**Review by codente — APPROVED** [review]
Looks good to me!

**Comment by nancyhild** [comment]
OK, NOW this is ready to look at again. Latest changes: extensive changes for terminology change from column layer -> column group

File: docs/beaver-builder/getting-started/bb-editor-basics/delete-a-module-or-column.md [hunk 1]
```diff
@@ -8,10 +8,7 @@ You can delete a module and leave the column, or you can delete a column,
 which also deletes the column's contents.
 
 :::tip **Tip**
-Deleting columns and child columns can get complicated and there is
-no Undo. To be on the safe side, save your work just before you delete so you
-can revert. It's also safer to delete modules first and then the empty
-columns.
+Deleting columns and nested columns can get complicated. It's  safer to delete modules first and then the empty columns and if you delete something you regret [use the Undo feature](/beaver-builder/getting-started/bb-editor-basics/undo-redo.md).
```
Comment (Dannyholt) [inline docs/beaver-builder/getting-started/bb-editor-basics/delete-a-module-or-column.md:11]:
```suggestion
Deleting columns and nested columns can get complicated. It's safer to delete modules first and then the empty columns and if you delete something you regret [use the Undo feature](/beaver-builder/getting-started/bb-editor-basics/undo-redo.md).
```

File: docs/beaver-builder/getting-started/what-can-i-do-with-beaver-builder.md [hunk 2]
```diff
@@ -82,16 +82,17 @@ Builder admin bar and then in the **Tools** menu click **Help**.
 
 When you drag content modules from the Content panel to the layout, the rows
 and columns are automatically created. As your layouts become more complex,
-with columns within columns, sometimes it's easier to drag in row layouts
-first, then drag content modules into them. For example, here is a single row
-with three layers of columns.
+with columns nested within columns, sometimes it's easier to drag in row layouts
```
Comment (Dannyholt) [inline docs/beaver-builder/getting-started/what-can-i-do-with-beaver-builder.md:85]:
```suggestion
with nested columns, sometimes it's easier to drag in row layouts
```

File: docs/beaver-builder/layouts/columns/column-layouts-overview.md [hunk 3]
```diff
@@ -4,112 +4,86 @@ title: Column layouts overview
...
 ---
 
-In Beaver Builder you cannot have rows within rows, but you can construct
-complex layouts with column layers and child columns.
+In Beaver Builder, you cannot place rows within rows, but you can construct
+complex column layouts within a single row, as shown in Figure 1.
 
-## Column layers, module layers, and child columns
+*Figure 1: An example of a complex column layout in one row*  
+![n example of a complex column layout in one row](/img/row-columns-column-layout-overview-5-nested.jpg)
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/columns/column-layouts-overview.md:11]:
```suggestion
![An example of a complex column layout in one row](/img/row-columns-column-layout-overview-5-nested.jpg)
```

File: docs/beaver-builder/layouts/columns/column-layouts-overview.md [hunk 4]
```diff
@@ -4,112 +4,86 @@ title: Column layouts overview
...
 
-![](/img/row-columns-column-layout-overview-1.png)
+*Figure 2: A schematic example of a complex column layout*  
+![ schematic example of a complex column layout](/img/columns-overview-column-schematic-example.png)
 
-Figure 1: One column with three module layers
+* A column group, shown with red outlines in Figure 2, is like a row within a row.  Each column group contains one or more columns arranged horizontally.
+This example has two main column groups and one nested column group.  
+The first column group has one column and the second column group has two columns, labeled 2a and 2b. 
+n Figure 2, Modules 7 and 8 are nested into a two-column group.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/columns/column-layouts-overview.md:23]:
```suggestion
In Figure 2, Modules 7 and 8 are nested into a two-column group.
```

File: docs/beaver-builder/layouts/columns/move-a-column.md [hunk 5]
```diff
@@ -4,9 +4,7 @@ title: Move a column
 sidebar_label: Move a column
 ---
 
-You can move both main and child columns and preserve the column style
-settings. When you move a child column, you have a choice to move just that
-column or the entire parent column to which it belongs.
+You can move both main and main columns and nested columns and preserve the column style settings. When you move a nested column, you have a choice to move just that column or the entire parent column to which it belongs.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/columns/move-a-column.md:7]:
```suggestion
You can move both main columns and nested columns and preserve the column style settings. When you move a nested column, you have a choice to move just that column or the entire parent column to which it belongs.
```

Comment (nancyhild) [reply docs/beaver-builder/layouts/columns/move-a-column.md]: Looks good.

File: docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md [hunk 6]
```diff
@@ -4,61 +4,60 @@ title: Tips for working with columns
...
+Using column groups and nesting columns can get, well, complicated. Here are some tips to help you get started. Along the way, you may find shortcuts that work better for you.
 
-## Tip #1: Understand the properties and limitations of using columns and
-child columns
+## Tip #1: Understand the properties and limitations of various column layouts
 
-See the [column layouts overview article](/beaver-builder/layouts/columns/column-layouts-overview.md). In
-particular, make sure you understand the concept of columns, module layers,
-and child columns.
+See the [column layouts overview article](/beaver-builder/layouts/columns/column-layouts-overview.md). In particular, make sure you understand the concept of column groups, nested columns, and multiple modules in a single column. See the [Responsive columns](#) article to understand how columns stack as screen width decreases.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md:11]:
```suggestion
See the [column layouts overview article](/beaver-builder/layouts/columns/column-layouts-overview.md). In particular, make sure you understand the concept of column groups, nested columns, and multiple modules in a single column. See the [Responsive columns](HERE) article to understand how columns stack as screen width decreases.
```

There is no link here

File: docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md [hunk 7]
```diff
@@ -4,61 +4,60 @@ title: Tips for working with columns
...
-![](/img/row-columns-tips-with-columns-3.png)
+![Two column groups or one parent column with two nested columns?](/img/column-groups-vs-nested-example.png)
 
-In some cases with a layer of columns, you'll find a module that isn't in its
-own column that you want to style individually. In this case, follow Tip #2 to
-drag in a row layout of a single column, then move that module into it. You
-may run up against the level of column nesting (see Tip #1) but may find
-another way to organize the column layers so you can get get the module into
-its own column.
+Just mousing over this layout, you can't tell if it's a row with two column groups (upper row and lower row) or a single column group with nested columns  for the buttons. Here's a diagram of the difference.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md:36]:
```suggestion
Just mousing over this layout, you can't tell if it's a row with two column groups (upper row and lower row) or a single column group with nested columns for the buttons. Here's a diagram of the difference.
```
Extra space.

File: docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md [hunk 8]
```diff
@@ -4,61 +4,60 @@ title: Tips for working with columns
...
 
-### Tip #4 Deleting modules is safer than deleting columns
+![Schematic of two column groups versus one column group with two nested columns](/img/column-groups-vs-nested-columns-schematic.png)
 
-When you mouse over **Delete column**, you can see the scope of what you'll
-be deleting. But since there's no Undo, it's even safer to delete modules
-first, then delete column structure. See [the article about deleting columns](/beaver-builder/getting-started/bb-editor-basics/delete-a-module-or-column.md).
+The best way to determine  a column's scope is to click the **Edit  Column** icon for a module, then mouse over **Column settings**. The blue shading that appears shows  the scope of the column containing that module.
```
Comment (Dannyholt) [inline docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md:40]:
```suggestion
The best way to determine  a column's scope is to click the **Edit  Column** icon for a module, then mouse over **Column settings**. The blue shading that appears shows the scope of the column containing that module.
```

Extra space.

File: docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md (hunk 6 above)
Comment (nancyhild) [reply docs/beaver-builder/layouts/columns/tips-for-working-with-columns.md]: Fixed and committed.

**Review by nancyhild — COMMENTED** [review]
OK think I got all the latest comments addressed.

**Review by Dannyholt — APPROVED** [review]

**Pull request merged**